ladders/Controllers/LaddersController.cs

```diff
@@ -0,0 +1,274 @@
+
+            var ladderModel = new LadderModel
+            {
+                MemberList = new List<ProfileModel>(),
+                CurrentRankings = new List<Ranking>(),
+                ApprovalUsersList = new List<ProfileModel>()
+            };
+
+
+            return View(ladderModel);
+        }
+
+
+        // POST: Ladders/Create
+        // To protect from overposting attacks, please enable the specific properties you want to bind to, for 
+        // more details see http://go.microsoft.com/fwlink/?LinkId=317598.
+        [HttpPost]
+        [ValidateAntiForgeryToken]
+        public async Task<IActionResult> Create([Bind("Id,Name,MemberList,CurrentRankings,ApprovalUsersList")]
+            LadderModel ladderModel)
+        {
+            if (!AmIAdmin()) return RedirectToAction(nameof(Index));
+
+            if (!ModelState.IsValid) return View(ladderModel);
+
+            _context.Add(ladderModel);
+            await _context.SaveChangesAsync();
+            return RedirectToAction(nameof(Index));
+        }
+
+        // GET: Ladders/Edit/5
+        public async Task<IActionResult> Edit(int? id)
+        {
+            if (!AmIAdmin()) return RedirectToAction(nameof(Index));
+
+            if (id == null) return NotFound();
+
+            var ladderModel = await _context.LadderModel.FindAsync(id);
+            if (ladderModel == null) return NotFound();
+            return View(ladderModel);
+        }
+
+        // POST: Ladders/Edit/5
+        // To protect from overposting attacks, please enable the specific properties you want to bind to, for 
+        // more details see http://go.microsoft.com/fwlink/?LinkId=317598.
+        [HttpPost]
+        [ValidateAntiForgeryToken]
+        public async Task<IActionResult> Edit(int id, [Bind("Id,Name,MemberList,CurrentRankings,ApprovalUsersList")]
+            LadderModel ladderModel)
+        {
+            if (!AmIAdmin()) return RedirectToAction(nameof(Index));
+
+            if (id != ladderModel.Id) return NotFound();
+
+            if (!ModelState.IsValid) return View(ladderModel);
+
+            try
+            {
+                _context.Update(ladderModel);
+                await _context.SaveChangesAsync();
+            }
+            catch (DbUpdateConcurrencyException)
+            {
+                if (!LadderModelExists(ladderModel.Id))
+                    return NotFound();
+                throw;
+            }
+
+            return RedirectToAction(nameof(Index));
+        }
+
+        // GET: Ladders/Delete/5
+        public async Task<IActionResult> Delete(int? id)
+        {
+            if (!AmIAdmin()) return RedirectToAction(nameof(Index));
+
+            if (id == null) return NotFound();
+
+            var ladderModel = await _context.LadderModel
+                .FirstOrDefaultAsync(m => m.Id == id);
+            if (ladderModel == null) return NotFound();
+
+            return View(ladderModel);
+        }
+
+        // POST: Ladders/Delete/5
```

Comment:
Instead of a POST to Ladders/Delete shouldn't this be DELETE to `Ladders/{id}` ?

Reply:
This is generated again.